app/controllers/api/v1/volunteers_controller.rb

```diff
@@ -0,0 +1,65 @@
+require 'net/http'
+module Api
+  module V1
+    class VolunteersController < ApplicationController
+      def show
+        volunteer = Volunteer.find(params[:id])
```

Comment:
This can be abstracted into a `find_volunteer` private method and reused in the update and delete actions. And you can add a before filter action, `before_action :find_user, only: %i[show, delete, update]` to run the method before those actions

Reply:
Done.